pkg/clients/edgeconnect/edgeconnect_test.go

```diff
@@ -0,0 +1,54 @@
+package edgeconnect
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMakeHostMappings(t *testing.T) {
+	hostPattern1 := "my-edgeconnect.dynatrace.a273ec656-603d-46c8-b5f5-5c47a6903dff.kubernetes-automation"
+	hostPattern2 := "super-edgeconnect.dynatrace.a273ec656-603d-46c8-b5f5-5c47a6903dfd.kubernetes-automation"
+
+	type args struct {
+		hostPatterns []string
+	}
+	tests := []struct {
+		name string
+		args args
+		want []HostMapping
+	}{
+		{
+			name: "No host patterns",
+			args: args{
+				hostPatterns: []string{},
+			},
+			want: []HostMapping{},
+		},
```

Comment:
I think we can add test case when:
```
args: args{
				hostPatterns: []string{defaultKubernetesDNS},
			},
```

Reply:
I had to refactor the whole function so this is deprecated. I will close the conversation.